apis/Google.Cloud.Diagnostics.AspNetCore/docs/index.md

```diff
@@ -181,7 +177,7 @@ can in inject the `IManagedTracer` into the action method using the `[FromServic
 
 ## Trace Outgoing HTTP Requests (recommended)
 
-The [recommended way of creating HttpClient](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-2.1) in ASP.NET Core 2.0 and upwards is to use the
+The [recommended way of creating HttpClient](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-2.1) in ASP.NET Core 2.1 and upwards is to use the
```

Comment:
I'd actually leave this - because the pattern *was* introduced in ASP.NET Core 2.0.